aphrodite/modeling/models/yi.py

```diff
@@ -166,31 +163,29 @@ class YiDecoderLayer(nn.Module):
     def __init__(
         self,
         config: YiConfig,
```

Comment:
Yi recently changed their config/modeling to make it completely based off of LLaMA - is a separate class still needed for this?